stock_picking_batch_grouped/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Manage product quantities in batch transfers.
```

Comment:
@SirAionTech Could you update a little bit this (as this is not clear at first sight) ?

Reply:
Thanks for having a look!
I explained what the new `Products` tab is, and added a few lines on how it works

Reply:
Maybe renaming the module `stock_picking_batch_grouped_by_product` or something like that should be great to reflect the functionality.

Reply:
While implementing your suggestion I realized/remembered that products is not the only thing the module is grouping by: it is also grouping by lot/serial, source location and destination location.

Moreover in the module it is possible to override some parts in order to customize the grouping, for instance the fields listed above are exposed in a dedicated method: https://github.com/OCA/stock-logistics-workflow/blob/3060c9ace643323cfda439275fab6c1dcb5cccad/stock_picking_batch_grouped/models/stock_picking_batch_line.py#L63-L70
I added a note about this in the README